app/controllers/search_results_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SearchResultsController < ApplicationController
+  def index
+
+    @suchEingabe = params[:search_query]
+
+    if !params[:search_query].empty?
```

Comment:
Wenn du erst hier `search_query` überprüfst, kannst du es auch direkt in Zeile 7 übergeben. Zuweisung ist hier ja dann nicht notwendig.

Reply:
anstelle von der zweiten params anfrage wird mit @suchEingabe die überprüfung gemach

Tests wurden hinzugefügt